gas/testsuite/gas/riscv/cv-march-rv32i-xcorev.s

```diff
@@ -0,0 +1,3 @@
+# Testing xcorev march option works for all CORE-V extensions
+target:
+	cv.starti 1, 320
```

Comment:
I think this test really ought to include all 6 HWLP instructions, to show that none are missed.